app/controllers/albums

```diff
@@ -0,0 +1,27 @@
+const rp = require('request-promise');
+const typicodePath = 'https://jsonplaceholder.typicode.com';
+
+const options = {
+    uri: '',
+    json: true, // Automatically parses the JSON string in the response
+    resolveWithFullResponse: false
+};
+
+const albums = (req, res, next) => {
+    options.uri = `${typicodePath}/albums`;
```

Comment:
I like how you used the `options` object to avoid reusing code, but try not to use mutable objects. They are not considered good practice as they are hard to follow and debug later on.
What you can do instead is define an options function, which takes the uri as a param and returns completed the `options` object.